tests/integration/test.replication.js

```diff
@@ -4248,6 +4248,8 @@ adapters.forEach(function (adapters) {
 // This test only needs to run for one configuration, and it slows stuff
 // down
```

Comment:
But... it's currently run for quite a few configurations.  And behaves very differently with different configurations.  Perhaps this comment should be updated.